docs/general/ad-filtering/create-own-filters.md

```diff
@@ -956,6 +960,13 @@ Blocking cookies and removing tracking parameters is achieved by using rules wit
 
 :::
 
+:::caution Restrictions
+
+- Modifier options should be lowercase, i.e. `$stealth=DPI` will be rejected.
+- Modifier options cannot be negated, i.e. `$stealth=~3p-cookie` will be rejected.
```

Comment:
Don't you think it makes the specification too verbose?
I mean why is this needed to be mentioned explicitly?

Reply:
As for me, this sounds complicated:
> Modifier options cannot be negated, i.e. `$stealth=~3p-cookie` will be rejected.

Reply:
But if a modifier supports it, it is explicitly stated in its description. Otherwise, you can assume it does not.

Reply:
I prefer to minimize the number of possible assumations